### PR TITLE
Feature/46 sub sidebar and sidebar new mockup ui

### PR DESCRIPTION
**Pull request by emekauja**
Screenshot 2022-10-27 at 4 54 32 PM

### CODE REVIEW

**Review by zsinnema — APPROVED** [review]

**Comment by zsinnema** [comment]
@ddelpiano do you want me to merge this one?

**Comment by ddelpiano** [comment]
> @ddelpiano do you want me to merge this one?

not yet, there are the changes we discussed yesterday to be made still. Thanks!

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -1,106 +1,182 @@
...
 import Node from './assets/svg/node.svg';
-import Cursor from './assets/svg/cursor.svg';
-import CursorActive from './assets/svg/cursor-active.svg';
-import Fullscreen from './assets/svg/fullscreen.svg';
-import FullscreenActive from './assets/svg/fullscreen-active.svg';
+import {
+  ControlIcon,
+  FileIcon,
+  HandIcon,
+  IntegratorIcon,
```
[the remark this answers is not in the document]
Comment (emekauja) [reply src/components/Sidebar.tsx]: fixes has been updated on sidebar node

File: src/components/Sidebar.tsx [hunk 2]
```diff
@@ -1,106 +1,159 @@
...
+export interface ISidebarNode {
+  id: string;
+  icon: React.ReactElement;
+  name: string;
+}
+
+const Sidebar = ({ sidebarNodes }: { sidebarNodes?: ISidebarNode[] }) => {
   const classes = useStyles();
   const [selected, setSelected] = React.useState('1');
+  const [subSelected, setSubSelected] = React.useState('Target');
```
Comment (ddelpiano) [inline src/components/Sidebar.tsx:58]: remove occurence of target from meta-diagram please

**Review by ddelpiano — CHANGES_REQUESTED** [review]

**Comment by ddelpiano** [comment]
@emekauja is this still required or can we close it?

**Pull request closed without merging**